### DISCUSSION TITLE
Improving application development in Vala

### DISCUSSION THREAD

**Discussion by Prince781** [elementary/edw#43] [Accepted Talks]
I will discuss ongoing work behind the scenes in improving the Vala developer experience, from work on the Vala Language Server to integrating this work in IDEs and with build systems, to internal improvements in the Vala compiler.

In addition, you will learn:
- about the various tools you can use to improve the quality of your Vala code and make you a more efficient developer
- about how to set up Vala tooling with your favorite editor so that, for example, you can get intelligent suggestions while you type

**Comment by Prince781** [comment]
Okay, I rewrote this a couple of times to better convey what I want this talk to be about. I expect it to be slanted towards discussing technical aspects of work on Vala tooling, but there will also be a tutorial component.

**Comment by cassidyjames** [comment]
@Prince781 I'm not sure if you saw, but we've accepted your talk. :) You can get started writing it now if you'd like, and we will provide further instructions/guidance around the recording once we've accepted all the talks.

**Reply by Prince781** [reply]
Okay, thanks for the heads up.
